content/en/docs/installation/installation-linux.md

```diff
@@ -20,30 +20,30 @@ Please modify the commands accordingly.
 
 {{< tabs >}}
 {{% tab name="Ubuntu" %}}
+
 ```bash
 git clone https://github.com/green-coding-berlin/green-metrics-tool ~/green-metrics-tool && \
 cd ~/green-metrics-tool && \
-git submodule update --init && \
```

Comment:
Why are you removing this?

Reply:
Submodules are now handled as part of the install script